src/parser.js

```diff
@@ -68,6 +69,11 @@ function findInDfa (a, obj) {
     return false;
 }
 
+// Add a comment
```

Comment:
our parser throws out comments? Does the cpython parser leave them in?

Reply:
If I recall, both Skulpt and cpython toss the comments out.

Reply:
But you keep them in here? But I don't see where you use them.

Reply:
Is there a better solution to parse comment ? I think this is not the optimal way